packages/react-charting/UnitTests/StackedBarChartUT.test.tsx

```diff
@@ -0,0 +1,186 @@
+import { StackedBarChartBase } from '../src/components/StackedBarChart/StackedBarChart.base';
+import { IChartDataPoint, IChartProps } from '../src/components/StackedBarChart';
+import { DefaultPalette } from '@fluentui/react';
+import { DarkTheme } from '@fluentui/theme-samples';
+const env = require('../config/tests');
+
+const runTest = env === 'TEST' ? describe : describe.skip;
+
+const points: IChartDataPoint[] = [
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '99%',
+  },
+  { legend: 'second', data: 1, color: DefaultPalette.green },
+];
+
+const areaLabelPoints: IChartDataPoint[] = [
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    xAxisCalloutData: '2020/04/30',
+  },
+  {
+    legend: 'first',
+    data: 3000000,
+    color: DefaultPalette.blue,
+    yAxisCalloutData: '50%',
+  },
+  {
+    data: 2000,
+    color: DefaultPalette.blue,
+  },
+];
+
+const data: IChartProps = {
+  chartTitle: 'Stacked Bar chart example',
+  chartData: points,
+};
+
+runTest('_createBarsAndLegends', () => {
+  test('Should return bars and legends which includes benchmark and target data', () => {
+    const instance = new StackedBarChartBase({
+      data: {},
```

Comment:
are you passing empty data here

Reply:
Yes Atishay while calling the actual function we are passing chart data. This is for instance creation only for thsi test